Documentation/configuration/metrics.rst

```diff
@@ -50,6 +53,8 @@ Policy Imports
 --------------
 
 * ``policy_count``: Number of policies currently loaded
+* ``policy_regeneration_seconds``: Total time taken to regenerate policies
```

Comment:
```
... Total sum of policy regeneration times
... Total sum of squares of policy regeneration times
```

Reply:
sure.

Reply:
done. Changed granularity to milliseconds for more realistic prometheus metrics gathering